pycalc11/interface.py

```diff
@@ -191,6 +190,23 @@ def run_driver(self):
 
         self._rerun = False
 
+    def interpolate_delays(self, t_0): 
+        """ Evaluates the delays at an absolute time t_0"""
+        x=(t_0-self.times[0]).sec 
+        return self.delays_dt(x)
+
+    @property
+    def delays_dt(self):
```

Comment:
Can you move this docstring to the `interpolate_delays` function, since that's more likely how this will be used?

Reply:
done!